docs/angular-integration.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[angular-integration]]
+=== Angular integration
+
+This document covers how to use Real User Monitoring JavaScript agent with Angular applications.
+
+[[installing-angular-integration]]
+==== Installing Elastic APM Angular package
+
+Install the `@elastic/apm-rum-angular` package as a dependency to your application:
+
+[source,bash]
+----
+npm install @elastic/apm-rum-angular --save
+----
+
+[float]
+==== Instrumenting your Angular application
+
+The Angular integration package comes with a `ApmService` that will start subscribing to 
+https://angular.io/api/router/Event[Angular Router Events] once the service is initialised. 
```

Comment:
s/initialised/initialized/ ?
(we should be consistent one way or the other; using z below)

Reply:
Good catch. "Initialized" is Standard American English (what we use in the docs) so this should be changed.

Reply:
Thanks @axw and @bmorelli25